flink-runtime/src/main/scala/org/apache/flink/runtime/messages/TaskManagerMessages.scala

```diff
@@ -113,4 +114,28 @@ object TaskManagerMessages {
    * @param cause reason for the external failure
    */
   case class FailTask(executionID: ExecutionAttemptID, cause: Throwable)
+  
+    // --------------------------------------------------------------------------
+  // Utility methods to allow simpler case object access from Java
+  // --------------------------------------------------------------------------
+  
+  def getNotifyWhenRegisteredAtJobManagerMessage() : AnyRef = {
+    NotifyWhenRegisteredAtJobManager
+  }
+  
+  def getRegisteredAtJobManagerMessage() : AnyRef = {
+    RegisteredAtJobManager
+  }
+  
+  def getRegisterAtJobManagerMessage() : AnyRef = {
+    RegisterAtJobManager
+  }
+
+  def getSendHeartbeatMessage() : AnyRef = {
+    SendHeartbeat
+  }
+
+  def getLogMemoryUsageMessage() : AnyRef = {
+    RegisteredAtJobManager
+  }
```

Comment:
As far as I know, it shouldn't compile to a variable and thus shouldn't make a difference for Java if you write a Scala function with or without parenthesis. At least IntelliJ's auto complete tells me that it is a function (referenced from Java code). It is only a Scala convention for pure functions.